### PR TITLE
One directory per block

### PR DESCRIPTION
**Pull request by simonhammes**
### Code Reviews and suggestions are much appreciated :)

### CODE REVIEW

File: src/blocks/email/index.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const { registerBlockType } = wp.blocks;
 import emailEdit from "./edit.js";
 import emailSave from "./save.js";
 import { getFieldTransform } from '../../block/functions';
-import { fieldParents } from '../../constants';
+import { fieldParents, myAttrs } from '../../constants';
```
Comment (simonhammes) [inline src/blocks/email/index.js]: This actually belongs to the previous commit :)
I simply forgot to import the 'myAttrs' constant

**Comment by simonhammes** [comment]
Update: resolved all merge conflicts

**Comment by ZafarKamal123** [comment]
Can you please resolve conflicts in this PR?

**Comment by simonhammes** [comment]
> Can you please resolve conflicts in this PR?

Done

**Pull request merged**